src/utils/leadtime_utils.py

```diff
@@ -28,3 +28,35 @@ def leadtime_months(start_month, leadtime):
         (list): List of all months across the leadtime
     """
     return [(start_month + i - 1) % 12 + 1 for i in range(leadtime)]
+
+
+def to_fc_month(pub_month, leadtime):
+    """
+    Given an input publication month and leadtime,
```

Comment:
Given how much we've discussed 0-indexing vs. 1-indexing leadtimes, maybe worth just mentioning here the convention that for a issue month of 1 and a leadtime of 0, the valid month is 1. Just so it's super clear and don't need to actually read the code.